zebra-consensus/src/primitives/halo2/tests.rs

```diff
@@ -91,7 +91,7 @@ fn generate_test_vectors() {
                             // FIXME: support OrchardZSA too, 580 works for OrchardVanilla only!
                             // FIXME: consider more "type safe" way to do the following conversion
                             // (now it goes through &[u8])
-                            enc_ciphertext: <[u8; 580]>::try_from(
+                            enc_ciphertext: <[u8; OrchardVanilla::ENCRYPTED_NOTE_SIZE]>::try_from(
```

Comment:
The type is already known
`enc_ciphertext: a.encrypted_note().enc_ciphertext.as_ref().try_into().unwrap().into(),`